packages/route-viewer-overlay/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@opentripplanner/route-viewer-overlay",
+  "version": "0.0.1",
+  "description": "A map overlay to show a transit route",
+  "main": "lib/index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/opentripplanner/otp-ui.git"
+  },
+  "author": "Evan Siroky",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/opentripplanner/otp-ui/issues"
+  },
+  "homepage": "https://github.com/opentripplanner/otp-ui#readme",
+  "dependencies": {
+    "@mapbox/polyline": "^1.1.0",
+    "@opentripplanner/core-utils": "^0.0.8",
+    "prop-types": "^15.7.2",
+    "react-leaflet": "^2.6.1"
```

Comment:
shouldn't prop-types and react-leaflet be peerDependencies, and base-map be a direct dependency?

Reply:
I think being too reliant on other packages to include these is unnecessary. I've only been doing this for packages that require one and only one version of a package to be installed. Otherwise, this should work fine.